trust-quorum/src/persistent_state.rs

```diff
@@ -18,9 +18,6 @@ use std::collections::BTreeMap;
 /// All the persistent state for this protocol
 #[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct PersistentState {
-    // Ledger generation
```

Comment:
There's no reason that the trust quorum protocol code should know how the persistent state is stored.